docs/mvc/dataBinding.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Data Binding
+name: Overview
+parent: Model View Controller
+nav_order: 2
+---
+
+# Make use Data Binding
```

Comment:
for this entire chapter alone I'd like to 🤗 you in the name of the entire UI5 dark matter developers;
still: "make use _of_ data binding"

Reply:
done